src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.Tests/GitFileManagerTests.cs

```diff
@@ -40,6 +40,11 @@ public class GitFileManagerTests
             "https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-4ac4c036/nuget/v3/index.json" })]
         [TestCase("WhiteSpaceCorrectlyFormatted", new string[] {
             "https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-7d57652f/nuget/v3/index.json" })]
+        [TestCase("NonManagedDisabledFeedSources", new string[0])]  // Don't do anything to normal disabled sources
+        [TestCase("RedisableCurrentlyEnabledIntSources", new string[] { // Flip enabled int source to disabled.
+            "https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-arcade-b0437974/nuget/v3/index.json" })]
+        [TestCase("EnsureAppendsDisableEntryAfterLastClear", new string[] { // Honor all existing disable entries and append after last found <clear/>
+            "https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-arcade-b0437974/nuget/v3/index.json" })]
```

Comment:
I think this case is not quite doing what the comment says. By adding it as a parameter to the test you're saying this is a new incoming managed feed, so the test is going to add it even if it didn't exist.

Reply:
I think basically, either the test should add it, and you should keep it here, or the test just wants to verify that existing entries will be added to the disabled list, in which case it should be removed from the input nuget.config

Reply:
Same applies to the Redisable test.

Reply:
Synced up. The redisable test is fine.

Reply:
- Clarified RedisableCurrentlyEnabledIntSources offline (it's updating the same feed that was already there, but enabled).  - Removed existing feed and added comments for EnsureAppendsDisableEntryAfterLastClear to explain its intent as well.